Allow access to response body from webhook request

magzot opened the pull request:
### Access response body in webhook requests.

With this PR, you can now reference the **response.body** inside your callback.

```
{
  "request": {
    "method": "POST",
    "url": "/add-job"
  },
  "response": {
    "status": 202,
    "body": "{\"jobId\":\"{{randomValue length=10 type='ALPHANUMERIC'}}\"}",
    "headers": {
      "Content-Type": "application/json"
    }
  },
  "serveEventListeners": [
    {
      "name": "webhook",
      "parameters": {
        "method": "POST",
        "url": "http://hostname:8080/add-job-callback",
        "headers": {
          "Content-Type": "application/json"
        },
        "body": "{ \"message\": \"success\", \"jobId\": \"{{jsonPath response.body '$.jobId'}}\" }"
      }
    }
  ]
}

```

If you think this implementation is on the right track, I'll proceed to finalise it with tests and documentation.

Cheers,
Magnus

## References

I think this is related to issue #2359.

- TODO

<!-- References to relevant GitHub issues and pull requests, esp. upstream and downstream changes -->

## Submitter checklist

- [ ] Recommended: Join [WireMock Slack](https://slack.wiremock.org/) to get any help in `#help-contributing` or a project-specific channel like `#wiremock-java`
- [X] The PR request is well described and justified, including the body and the references
- [ ] The PR title represents the desired changelog entry
- [ ] The repository's code style is followed (see the contributing guide)
- [ ] Test coverage that demonstrates that the change works as expected
- [ ] For new features, there's necessary documentation in this pull request or in a subsequent PR to [wiremock.org](https://github.com/wiremock/wiremock.org)

<!--
Put an `x` into the [ ] to show you have filled the information.
The template comes from https://github.com/wiremock/.github/blob/main/.github/pull_request_template.md 
You can override it by creating .github/pull_request_template.md in your own repository 
-->

magzot committed:
Allow access to response body from webhook request

leeturner commented:
Hi, many thanks for your contribution.  There are a couple of things that I was wondering if you could take a look at.  When we use the original request in the webhook we specifically map this to the `originalRequest` key.  To maintain consistency we should map the response template model to `originalResponse`.

Also, there are no tests in your PR.  We don't merge PRs without an acceptable level of test coverage of your change.   If you could add tests that would be great.

magzot committed:
Merge branch 'wiremock:master' into master

magnus-zotterman-wcar committed:
Merge remote-tracking branch 'upstream/master'

magnus-zotterman-wcar committed:
Align webhook response templating with originalRequest

Expose the triggering response to webhook templates as `originalResponse`
instead of `response`, matching the existing `originalRequest` naming.
